tests/mitol/payment_gateway/test_cybersource.py

```diff
@@ -5,8 +5,10 @@
 from typing import Dict
```

Comment:
Could you move this file into a `api/` directory? I think that makes it clearer which module in `mitol.payment_gateway` it's testing.